kaltura/kaltura-interactive-player
# regenerate package-lock.json file to fix npm install issues

# Before
Failed to execute `npm i`

# Changes
- regenerated package-lock.json file. 

# After
can execute `npm i`

# How to test
1. delete folder `node_modules`
2. run `npm i`
3. check that app works as expected


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/kaltura/kaltura-interactive-player/6)
